docs/minio-admin-complete-guide.md

```diff
@@ -608,23 +608,40 @@ USAGE:
   mc admin config COMMAND [COMMAND FLAGS | -h] [ARGUMENTS...]
 
 COMMANDS:
-  get     get config of a MinIO server/cluster.
-  set     set new config file to a MinIO server/cluster.
+  get      interactively retrieve a config key parameters
+  set      interactively set a config key parameters
+  reset    interactively reset a config key parameters
+  history  show all historic configuration changes
+  restore  rollback back changes to a specific config history
+  export   export all config keys to STDOUT
+  import   import multiple config keys from STDIN
 
 FLAGS:
   --help, -h                       Show help.
 ```
 
-*Example: Get server configuration of a MinIO server/cluster.*
+*Example: Get specific settings on 'etcd' sub-system.*
```

Comment:
```suggestion
*Example: Get 'etcd' sub-system configuration.*
```

Maybe it is just me, but to me `specific settings on 'etcd' sub-system` sounds like user can get a specific setting or a configuration parameter of `etcd`, like:

```
$ mc admin config get myminio etcd path_prefix`
etcd path_prefix= 'prefix/'
```
but this is not valid command format, so it fails with an error message.